### DISCUSSION TITLE
智慧枕頭助你一覺到天亮🛌

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#843] [General]
**新聞來源:**
https://buzzorange.com/techorange/2022/03/14/zerema-sleep/
**新聞摘要:**
韓國 Zerema 枕頭結合 AI 深度學習來紀錄使用者的睡眠習慣，該枕頭配備 AI 驅動的打呼檢測技術，當枕頭感覺到使用者打呼時，就會自己提高或降低枕頭的高度，調整適當的呼吸形式，提供個性化的睡眠分析。
**心得:**
這個枕頭感覺還不錯，想不到連枕頭這種東西都可以結合科技!
但 但是價格一個快一萬塊，看來只能遠觀不敢下手購買了
(或是等有錢了買一組給父母)
![image](https://user-images.githubusercontent.com/90883526/201512615-223663a8-8ba6-4bfd-a266-bda0172eaf16.png)

**Comment by ru0372572** [comment]
酷枕頭...現代人總是為睡眠所困擾，AI應用真的無所不在!!

**Comment by Kotori0501** [comment]
好貴不太敢下手
我本身就是睡不太著的人，有了這個應用於AI技術的枕頭，不知道會不會讓我的睡眠品質變好

**Comment by moonnight3503** [comment]
太神奇了!竟然連打呼聲都能夠做AI數據分析的資料，還能夠幫使用者調整枕頭高度，真是開了我的眼界!

**Comment by a45320614** [comment]
利用AI分析睡眠習慣感覺是個很實際的應用
這個枕頭應該可以讓我睡好一點
早上的課不會再睡過頭了

**Comment by wenchi9023** [comment]
現代人生活壓力大，有這顆枕頭確實能改善人們生活品質，是個很符合時代的科技應用!!
不過似乎也只適合那些睡覺不會移位的人，否則就有失這顆枕頭的意義了!

**Comment by Chiway1229** [comment]
有好枕頭才能擁有良好的學習品質
我建議學校給每個學生發一個

**Reply by chia-kai-chang** [reply]
同學有沒有興趣來創業~ 😂

**Comment by peggy0329** [comment]
比起智慧枕頭，我覺得我更需要智慧抱枕哈哈哈
每次出遊時，睡在不習慣的地方總是會睡不好，如果有智慧抱枕可以配合使用者的個人習慣播放放鬆音樂等功能！
感覺真的很不錯！

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
好的睡眠對現代人來說真的很奢侈 Ai的這項發明對人類授益良多

**Comment by StevenWanglolz** [comment]
好的睡眠真的對人有很大的影響，沒想到竟然有人把枕頭和AI科技結合在一起，真的令人佩服! 不過這價格有點...

**Reply by bonny123865** [reply]
價格不太友善，看到之後直接心想
這根本是為有錢人做的設計!

**Comment by Evan0979** [comment]
一覺到天亮嗎...前提是要先睡得著诶
我比較需要能個快速入睡的方法QQ

**Comment by williamyen0623** [comment]
最近真的都找不到好用的枕頭，躺習慣的枕頭也舊了，如果沒有預算考量的話，我或許會參考一下這顆酷酷的枕頭

**Comment by manipulateed** [comment]
感覺很適合我欸，我超容易因為鼻中隔微彎曲造成晚上睡覺都會打呼或是吸不太到什麼空氣，而且吸不到氣只會越睡越累，睡眠是人很重要的一部分，如果這能解決這個問題價格算是很值得的吧🤔

**Comment by Zoe06060606** [comment]
這枕頭我們大學生真的很需要 尤其壓力大的時後 好好睡一覺很重要

**Comment by CPCpyt** [comment]
好酷，希望未來能買一個給自己，感覺能有很棒的體驗！
